Add PSA Signal Registration Blog (#10)
privacyguides/blog.privacyguides.org: docs/posts/signal-number-registration-update.md

---
date: 2022-11-10
categories:
    - Software
authors:
    - matchboxbananasynergy
tags:
    - PSA
    - Signal
    - Instant Messengers
links:
    - Signal Configuration Guide: https://www.privacyguides.org/real-time-communication/signal-configuration-hardening/
    - Real-Time Communication: https://www.privacyguides.org/real-time-communication/
license: BY-SA
---
# Important Changes to Signal Registration and Registration Lock

> There has been a recent change to how Signal handles registration which primarily affects people who are using a number that they don't have exclusive access to for Signal.

Reviewer: Signal has changed how it handles registration. This primarily affects people who are using a number that they don't have exclusive access to for Signal.

## How It Used to Work

> As outlined in our previous [Signal Configuration & Hardening Guide](https://blog.privacyguides.org/2022/07/07/signal-configuration-and-hardening/#signal-pin), if you register a number with Signal, set up Registration Lock and check into the app at least once every 7 days, nobody could use the number you've claimed and try to re-register it for themselves without knowing your Registration Lock PIN.

Reviewer: In our [previous](https://blog.privacyguides.org/2022/07/07/signal-configuration-and-hardening/#signal-pin) guide we outlined how to prevent people re-registering your number for Signal using the Registration Lock feature.

>If Alice registers number X and enables reglock, but Bob later proves ownership of number X (by registering and completing the SMS code), then Alice will >be unregistered. However, if a reglock is present, Bob still won't be able to register immediately if he does not know the reglock code. This allows >reglock to still function as a way to prevent someone else from taking over your account.
>
>However, by unregistering Alice, this starts a 7-day timer. After 7 days, if Alice doesn't re-register, then the reglock is removed and Bob will be free >to register the number without needing to know the reglock. But if Alice still truly does own the number, she can simply re-prove ownership and things >should go back to normal for her.
>
>This is important because phone number can (and are) re-used among cell carriers. If someone gets a new phone number from their carrier, they should not >be prevented from registering with Signal indefinitely because the previous owner has reglock.
>
>The intention of reglock is to prevent hijacking of numbers you actually own, not to guarantee the number for yourself for life.

> While this change makes sense from the perspective of making it so you cannot "hold a number hostage" as long as you keep checking in, it is particularly important for people who've used disposable phone numbers to know this. We recommend migrating to a phone number that you own and will be able to own for the foreseeable future in order to avoid getting locked out of your account and losing your contacts.

Reviewer: hile this change makes sense from the perspective of making it so you cannot "hold a number hostage" as long as you keep checking in, it is particularly important for people who've used disposable phone numbers to know this. 

We recommend migrating to a phone number that you own and will be able to own for the foreseeable future in order to avoid getting locked out of your account and losing your contacts.

---
date: 2022-11-10
categories:
    - Software
authors:
    - matchboxbananasynergy
tags:
    - PSA
    - Signal
    - Instant Messengers
links:
    - Signal Configuration Guide: https://www.privacyguides.org/real-time-communication/signal-configuration-hardening/
    - Real-Time Communication: https://www.privacyguides.org/real-time-communication/
license: BY-SA
---
# Important Changes to Signal Registration and Registration Lock

Signal has changed how it handles registration. This primarily affects people who are using a number that they don't have exclusive access to for Signal.

## How It Used to Work

> As outlined in our previous [Signal Configuration & Hardening Guide](https://blog.privacyguides.org/2022/07/07/signal-configuration-and-hardening/#signal-pin), if you register a number with Signal, set up Registration Lock and check into the app at least once every 7 days, nobody could use the number you've claimed and try to re-register it for themselves without knowing your Registration Lock PIN.

Reviewer 2: ```suggestion
As outlined in our previous [Signal Configuration & Hardening Guide](https://blog.privacyguides.org/2022/07/07/signal-configuration-and-hardening/#signal-pin), if you register a number with Signal, set up Registration Lock and check into the app at least once every 7 days, nobody could use the number you'd claimed and try to re-register it for themselves without knowing your Registration Lock PIN.
```

---
date: 2022-11-10
categories:
    - Software
authors:
    - matchboxbananasynergy
tags:
    - PSA
    - Signal
    - Instant Messengers
links:

>     - Signal Configuration Guide: https://www.privacyguides.org/real-time-communication/signal-configuration-hardening/

Reviewer 3: this doesn't look right, i think other blog posts use internal links (i.e. `.md` links) for internal related links

Reviewer 3: Yeah but that page isn't on www.privacyguides.org anymore, so it should be:

```suggestion
    - Signal Configuration Guide: signal-configuration-and-hardening.md
```

    - Real-Time Communication: https://www.privacyguides.org/real-time-communication/
license: BY-SA
---
# Important Changes to Signal Registration and Registration Lock

Signal has changed how it handles registration. This primarily affects people who are using a number that they don't have exclusive access to for Signal.

## How It Used to Work

> As outlined in our previous [Signal Configuration & Hardening Guide](https://blog.privacyguides.org/2022/07/07/signal-configuration-and-hardening/#signal-pin), if you registered a number with Signal, set up Registration Lock and checked into the app at least once every 7 days, nobody could use the number you'd claimed and try to re-register it for themselves without knowing your Registration Lock PIN.

Reviewer 3: ```suggestion
As outlined in our previous [Signal Configuration & Hardening Guide](signal-configuration-and-hardening.md#signal-pin), if you registered a number with Signal, set up Registration Lock and checked into the app at least once every 7 days, nobody could use the number you'd claimed and try to re-register it for themselves without knowing your Registration Lock PIN.
```

same issue...

---
date: 2022-11-10
categories:
    - Software
authors:
    - matchboxbananasynergy
tags:
    - PSA
    - Signal
    - Instant Messengers
links:

>     - Signal Configuration Guide: signal-configuration-and-hardening.md

Reviewer 3: oops my bad, it's supposed to be like https://github.com/privacyguides/blog.privacyguides.org/blob/main/docs/posts/firefox-privacy.md

```suggestion
    - posts/signal-configuration-and-hardening.md
```
